SIGNUP.html

```diff
@@ -0,0 +1,260 @@
+        border: none;
+      }
+
+      .card1 {
+        width: 50%;
+        padding: 40px 30px 10px 30px;
+      }
+
+      .card2 {
+        width: 50%;
+        background-image: linear-gradient(130deg, #183a36 40%, #05272e 100%);
+      }
+
+      #logo {
+        width: 70px;
+        height: 60px;
+      }
+
+      .heading {
+        margin-bottom: 60px !important;
+      }
+
+      ::placeholder {
+        color: #000 !important;
+        opacity: 1;
+      }
+
+      :-ms-input-placeholder {
+        color: #000 !important;
+      }
+
+      ::-ms-input-placeholder {
+        color: #000 !important;
+      }
+
+      .form-control-label {
+        font-size: 12px;
+        margin-left: 15px;
+      }
+
+      .msg-info {
+        padding-left: 15px;
+        margin-bottom: 30px;
+      }
+
+      .btn-color {
+        border-radius: 50px;
+        color: #fff;
+        background-image: linear-gradient(130deg, #05272e 40%, #183a36 100%);
+        padding: 15px;
+        cursor: pointer;
+        border: none !important;
+        margin-top: 40px;
+      }
+
+      .btn-color:hover {
+        color: #fff;
+        background-image: linear-gradient(130deg, #183a36 40%, #05272e 100%);
+      }
+
+      .btn-white {
+        border-radius: 50px;
+        color: #d500f9;
+        background-color: #fff;
+        padding: 8px 40px;
+        cursor: pointer;
+        border: 2px solid #277508 !important;
+      }
+
+      .btn-white:hover {
+        color: #fff;
+        background-image: linear-gradient(130deg, #05272e 40%, #183a36 100%);
+      }
+
+      a {
+        color: #000;
+      }
+
+      a:hover {
+        color: #000;
+      }
+
+      .bottom {
+        width: 100%;
+        margin-top: 50px !important;
+      }
+
+      .sm-text {
+        font-size: 15px;
+      }
+
+      .ml-2 a:hover {
+        color: white;
+      }
+
+      @media screen and (max-width: 992px) {
+        .card1 {
+          width: 100%;
+          padding: 40px 30px 10px 30px;
+        }
+
+        .card2 {
+          width: 100%;
+        }
+
+        .right {
+          margin-top: 100px !important;
+          margin-bottom: 100px !important;
+        }
+      }
+
+      @media screen and (max-width: 768px) {
+        .container {
+          padding: 10px !important;
+        }
+
+        .card2 {
+          padding: 50px;
+        }
+
+        .right {
+          margin-top: 50px !important;
+          margin-bottom: 50px !important;
+        }
+      }
+    </style>
+    <script
+      type="text/javascript"
+      src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"
+    ></script>
+    <script
+      type="text/javascript"
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.bundle.min.js"
+    ></script>
+    <script type="text/javascript"></script>
+  </head>
+  <body oncontextmenu="return false" class="snippet-body">
+    <div class="container px-4 py-5 mx-auto">
+      <div class="card card0">
```

Comment:
No styles present with `.card0` so it can be removed.

Reply:
YUP that also done